### PR TITLE
Task/create view dashboard

### PR DESCRIPTION
**Pull request by erikiado**
#### What's this PR do?
It creates the main dashboard panel and the user panel which shows the available users.
#### Where should the reviewer start?
administracion/urls.py
administracion/views.py
administracion/tests.py
administracion/test_views.py
jp2_online/templates/administracion/
jp2_online/templates/base/

#### How should this be manually tested?
python manage.py test administracion

#### Any background context you want to provide?
N/A

This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)

### CODE REVIEW

File: administracion/test_views.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        """
+        test_url_name = 'administracion:administracion_principal'
+        self.browser.visit(self.live_server_url + reverse(test_url_name))
+
+        # Check for nav_bar partial
+        self.assertTrue(self.browser.is_text_present('Instituto Juan Pablo'))
+        # Check for side_nav partial
+        self.assertTrue(self.browser.is_text_present('Administración'))
+
+    def test_panel_usuarios(self):
```
Comment (AndreeToledo) [inline administracion/test_views.py]: I think this test should be more robust, creating at leat 5 users, each with different roles, and at least one Capturista

File: administracion/tests.py [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
-# Create your tests here.
+
+class TestAdministracion(TestCase):
+    """Unit test suite for testing the views in the app: administracion.
+
+    Test that the views for 'administracion' are correctly received as a response and that
+    they use the correct template.
+    """
+
+    def test_view_panel_principal(self):
```
Comment (AndreeToledo) [inline administracion/tests.py]: This tests are really cool 👍

File: administracion/urls.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from django.conf.urls import url
+from .views import admin_panel_principal, admin_panel_usuarios
+
+app_name = 'administracion'
+
+urlpatterns = [
+    url(r'^panel/', admin_panel_principal, name='administracion_principal'),
```
Comment (AndreeToledo) [inline administracion/urls.py]: Me parece que el poner el administración en el name, es algo redundante y nulifica el efecto de usar ```app_name = 'administracion'```

Por que ```administracion:principal``` es mejor que ```adminsitracion:administracion_principal```

File: administracion/tests.py [hunk 4]
```diff
@@ -1,3 +1,27 @@
...
+        '''Unit Test: administracion.views.admin_panel_principal.
```
Comment (AndreeToledo) [inline administracion/tests.py]: Doctsrings should always be delimited by double quotes.

File: administracion/tests.py [hunk 5]
```diff
@@ -1,3 +1,27 @@
...
+        '''
+        # self.client.login(username='user', password='test')
+        test_url_name = 'administracion:administracion_principal'
+        response = self.client.get(reverse(test_url_name), follow=True)
+        self.assertEqual(200, response.status_code)
+        self.assertTemplateUsed(response, 'administracion/dashboard_main.html')
+
+    def test_view_panel_usuarios(self):
+        '''Unit Test: administracion.views.admin_panel_usuarios.
```
Comment (AndreeToledo) [inline administracion/tests.py]: Docstrings should always be delimited by double quotes.

File: administracion/views.py [hunk 6]
```diff
@@ -1,3 +1,15 @@
-# from django.shortcuts import render
+from django.contrib.auth.models import User
+from django.shortcuts import render
 
-# Create your views here.
+
+def admin_panel_principal(request):
+    '''View to render the main control dashboard.
```
Comment (AndreeToledo) [inline administracion/views.py]: Docstrings should always be delimited by double quotes.

File: administracion/urls.py (hunk 3 above)
Comment (Estebes10) [inline administracion/urls.py]: I think the same

File: administracion/urls.py [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+    url(r'^usuarios/', admin_panel_usuarios, name='administracion_usuarios')
```
Comment (Estebes10) [inline administracion/urls.py]: Same comment as in the above line: administracion:usuarios

**Pull request merged**